doc/instances.md

```diff
@@ -826,11 +826,17 @@ Key         | Type      | Default           | Required  | Description
 :--         | :--       | :--               | :--       | :--
 vendorid    | string    | -                 | no        | The vendor id of the USB device
 productid   | string    | -                 | no        | The product id of the USB device
+portglob    | string    | -                 | no        | Glob matching to physical USB port
```

Comment:
I suspect `port` or `portpath` would be clearer. I don't know how common `glob` is with our users.

I'd also have the description be something like `USB port path (wildcard supported) (e.g. 9/3/1)`